.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.suggest.jsdoc.generateReturns": false
+}
```

Comment:
is it needed?

Reply:
I don't think that should be add to repo, but I'll use it locally, ty